content/concepts/targets.md

```diff
@@ -1,10 +1,80 @@
 ---
 title: Targets
 ---
-> webworker
-> node
-> async-node
-> node-webkit
-> electron
-> electron-main
-> electron-renderer
+## Overview
+
+Because JavaScript can be written for both server & browser, *webpack* offer's multiple deployment _targets_.
+
+*webpack* offer's a variety of deployment _targets_ that you can set in your webpack [configuration](./api/configuration).
+
+W> The webpack `target` property is not to be confused with the `output.target` property. For more information see [our guide](./concepts/output) on the `output` property.
+
+
+### Usage
+
+To set the `target` property, you simply set the target value in your webpack config: 
+
+```javascript
+// webpack.config.js
+
+module.exports = config;
+
+config = {
+  target: 'node' // <===== Just pass the name of the target  
```

Comment:
You can skip "just". Often it doesn't add value and it's better to leave it out.